src/libs/ReportUtils.js

```diff
@@ -602,7 +602,13 @@ function getOldDotDefaultAvatar(login = '') {
  * @returns {Boolean}
  */
 function isDefaultAvatar(avatarURL) {
-    if (_.isString(avatarURL) && (avatarURL.includes('images/avatars/default-avatar_') || avatarURL.includes('images/avatars/user/default'))) {
+    if (_.isString(avatarURL)
```

Comment:
I broke this down on multiple lines for the linter. I followed the same pattern we do [here](https://github.com/Expensify/App/blob/4ff8ce56ad0960a77f703c5651b2c537062f10a3/src/components/AttachmentModal.js#L117-L126)

Reply:
TBH, I think it is a stupid linter rule for code. I understand it for comments, but code should be able to be longer than 190 characters per line. But...that's a topic for a different day and time. :)